components/Header.js

```diff
@@ -15,6 +15,7 @@ export default function Header() {
           src={waxGreenLogo}
           alt="WAX logo"
           style={{ cursor: 'pointer' }}
+          priority
```

Comment:
Just been reading up on this property. Out of interest was it the linter that raised this optimisation? If so that's handy!

Reply:
Yeah! I was trying to get rid of all the errors/warnings that were thrown in the console when the page was loaded in dev mode and this was one of them.